lexos/static/js/scripts_statistics.js

```diff
@@ -207,9 +216,13 @@ $(function () {
         generateStatsBoxPlot()
         // Display the result.
         $('#corpus-stats-result').css({'display': 'block'})
+        $('html, body').animate({
+          scrollTop: $('#get-stats').offset().top
+        }, 1000)
       } else { // Else hide the corpus stats result div.
-        $('#corpus-stats-result').css({'display': 'none'})
+        $('#file-stats-result').css({'display': 'none'})
```

Comment:
Why does this line needs to get changed?

Reply:
MY BAD. This absolutely does not need to be changed. That was an accident left over from testing something.